code/index.html

```diff
@@ -4,18 +4,47 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project Name</title>
-    <!-- dont forget to add a css file and link it here! -->
-    <!-- If you're using google fonts, that should be added here -->
+    <title>Spis meg</title>
+    <!-- dont forget to add a css file and link it here! --> <link rel="stylesheet" href="style.css">
+    <!-- If you're using google fonts, that should be added here --> 
   </head>
-  <body>
+
+  <body> 
     <!-- 
       Write your code here inside the body to make it appear in the browser. 
       Below is an example on how to use an image tag. Copy it and add it in the body, but outside the comment tag if you want to see how it looks like on the website. Then you can change the size and other styling for it in the css file.
 
       <img src="example-recipe-background.png" alt="Banana Bread">
 
     -->
-    <h1>Hello there! 👋🏼</h1>
+      <img class="main-image" src="image.food.jpg" alt="food">
+
+<div> 
+         <h1>SPIS MEG</h1>
+
+      <h2>Kaster du ofte mat? Synes du det er vanskelig å utnytte all maten du har i kjøleskapet?</h2h>
+      
+      <h3>Spis Meg er det digitale oppslagsverket som foreslår oppskrifter basert på hvilke matvarer du allerede har i kjøleskapet</h3>
+
+      <h3>Hvorfor? Hver nordmann kaster gjennomsnittlig 42 kg mat i året. Norge har forpliktet seg til FNs bærekraftmål, og skal innen 2030 halvere matsvinnet per innbygger.
+      Kast mindre mat, spar penger, og gjør noe godt for kloden vår.</h3> 
+
+      
+
+    </div>
```

Comment:
We should indent our code to make it easier to read. Take a look at our [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e) if you wanna learn more :)

```
<img class="main-image" src="image.food.jpg" alt="food">
<div> 
  <h1>SPIS MEG</h1>
  <h2>Kaster du ofte mat? Synes du det er vanskelig å utnytte all maten du har i kjøleskapet?</h2h>
  <h3>Spis Meg er det digitale oppslagsverket som foreslår oppskrifter basert på hvilke matvarer du allerede har i kjøleskapet</h3>
  <h3>Hvorfor? Hver nordmann kaster gjennomsnittlig 42 kg mat i året. Norge har forpliktet seg til FNs bærekraftmål, og skal innen 2030 halvere matsvinnet per innbygger.
      Kast mindre mat, spar penger, og gjør noe godt for kloden vår.</h3> 
</div>

```